libraries/ashrae/guideline36/4.10-chilled-water-plant.ttl

```diff
@@ -0,0 +1,1221 @@
+    # optional
+    sh:property [
+        rdfs:label "CH CHW isolation valve closed end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "CH CHW isolation valve open end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Off_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # Retain the following points for optional analog valve position feedback.
+    # optional
+    sh:property [
+        rdfs:label "CH CHW isolation valve position feedback" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Valve_Position_Sensor ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+# TODO see comment
+:ch-chw-bypass-valve a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Chiller CHW Bypass Valve" ;
+    rdfs:comment "Required for series piped chillers and primary-only parallel CHW plants with waterside economizers. Delete otherwise." ;
+    sh:class brick:Bypass_Valve ;
+    # applications
+    sh:property [
+        rdfs:label "CH CHW bypass valve" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Off_Command ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMinCount 1
+    ] .
+
+# TODO
+# Required for plants with headered condenser water pumps. Delete otherwise. CW isolation valves may be two-position
+# for chillers that do not require head pressure control or for plants with variable speed condenser
+# pumps but no waterside economizer.
+:ch-cw-isolation-valve a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Chiller CW Isolation Valve" ;
+    sh:class brick:Condenser_Water_Isolation_Valve ;
+    # applications, required if present
+    sh:property [
+        rdfs:label "CH CW isolation valve" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [
+            sh:or (
+                sh:class brick:Valve_Position_Command
+                sh:class brick:On_Off_Command
+            )
+        ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # Retain the following two points for optional valve end switch status feedback.
+    # optional
+    sh:property [
+        rdfs:label "CH CW isolation valve closed end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "CH CW isolation valve open end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Off_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
```

Comment:
Usually we use open/Close enumeration for the Valve and damper, and on/off is for the Fan and pumps. And the Brick https://brickschema.org/ontology/1.3/classes/Open_Close_Status will show the current valve status (open/closed)

Reply:
Ok. The G36 points are `status` points, with a note that reads...
>Retain the following two points for optional valve end switch status feedback.

Keeping as is since these are two distinct points.